engine/class_modules/sc_death_knight.cpp

```diff
@@ -3015,9 +3015,12 @@ struct death_knight_action_t : public Base
         {
           continue;
         }
-        action_t::sim->out_log.printf("%s spreads shackle the unworthy with %s from %s to target %s (remains=%.3f)",
-                action_t::player->name(), this->name(), action_t::target->name(), destination->name(),
-                source_dot->remains().total_seconds() );
+        if ( action_t::sim->log )
+        {
+          action_t::sim->out_log.printf("%s spreads shackle the unworthy with %s from %s to target %s (remains=%.3f)",
+                  action_t::player->name(), this->name(), action_t::target->name(), destination->name(),
+                  source_dot->remains().total_seconds() );
+        }
```

Comment:
Although this works, I encourage you to switch to sim -> print_log() that contains the sim -> log check and uses a slightly different (and simpler) formatting:
```suggestion
        sim -> print_log("{} spreads shackle the unworthy with {} from {} to target {} (remains={} )",
                  player->name_str, name_str, target->name_str, destination->name_str,
                  source_dot->remains().total_seconds() );
        }
```